test/dos.jl

```diff
@@ -0,0 +1,27 @@
+@testitem "density of states (dos, ldos) computation for silicon" #=
+    =#    tags=[:dont_test_mpi] setup=[TestCases] begin
+    using DFTK
+    testcase = TestCases.silicon
+
+    atoms=fill(ElementPsp(testcase.atnum; psp=load_psp(testcase.psp_upf)), 2)
+	model = model_LDA(testcase.lattice, atoms, testcase.positions; temperature=1e-2)
+	basis = PlaneWaveBasis(model; Ecut=12, testcase.kgrid)
+	scfres = self_consistent_field(basis, tol=1e-8);
+
+	plot_dos(scfres)
+	plot_ldos(scfres)
```

Comment:
Ok, but that's not really a test. That's just running it.

As soon as you have an example, it will also be run automatically during testing, so this is no longer needed. Either you test something about the computation here or you just drop the test.